loader/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ if(WIN32)
         endif()
     endif()
     option(USE_MASM "Use MASM" ON)
-    if (USE_MASM)
+    if ((MSVC OR JWASM_FOUND) AND USE_MASM)
```

Comment:
Why not just set USE_MASM to OFF like this?

```
if (MSVC OR JWASM_FOUND)
    set(USE_MASM OFF)
endif()
option(USE_MASM "Use MASM" ON)
```

This is much more clear.

Reply:
Actually just setting `USE_MASM` to `OFF` would work.

Like so:

```
        find_program(JWASM_FOUND jwasm)
        if (JWASM_FOUND)
            set(USE_MASM OFF)
            set(CMAKE_ASM_MASM_COMPILER ${JWASM_FOUND})
```

Reply:
Went with the first approach.